Demo-Android/Gutenberg/src/main/java/org/wordpress/gutenberg/GutenbergRequestInterceptor.kt

```diff
@@ -1,13 +1,15 @@
 package org.wordpress.gutenberg
 
 import android.webkit.WebResourceRequest
+import android.webkit.WebResourceResponse
+import android.webkit.WebView
 
 public interface GutenbergRequestInterceptor {
-    fun interceptRequest(request: WebResourceRequest): WebResourceRequest
+    fun modifyRequest(view: WebView, request: WebResourceRequest): WebResourceResponse?
```

Comment:
It's not clear to me where `view` is used – is this just an iOS-style "delegates should receive a reference to the object they're operating on in case there's more than one of them" thing? 

Or did I miss something? 😁

Reply:
The inclusion of `view` was solely mirroring the function signature of [`shouldInterceptRequest`](https://developer.android.com/reference/android/webkit/WebViewClient#shouldInterceptRequest(android.webkit.WebView,%20android.webkit.WebResourceRequest))—i.e., delegating the entirely of the hook capability to the delegate. 

The `view` parameter is likely unnecessary for this particular use case and could be removed to simplify the function and limit its abilities.

Reply:
Removed in 946b8c24e4e55e42304fa0821d27307915ef9cb8.